client/src/app-storage.ts

```diff
@@ -4,8 +4,6 @@ import {Hexalot} from "./island/hexalot"
 import {Island, IslandPattern} from "./island/island"
 import {Journey} from "./island/journey"
 
-const MASTER_KEY = "master"
```

Comment:
What replaces the concept of "Master"?

Reply:
Nothing for now.  I can haz Twitter handle?